### PR TITLE
add about page

### LINKED ISSUE #160: About
# Problem

Currently the **About** menu item is a dead link. We would like this to give us a short project summary.

# Proposed Solution

Add a new page with the description text.

### PR DESCRIPTION
**Pull request by ann26**
fix #160 

screencapture-0-0-0-0-5000-campaign_manager-about-1499934428053

### CODE REVIEW

**Comment by dimasciput** [comment]
@ann26 can you fix the travis error?

File: flask_project/campaign_manager/templates/about.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    .page-content {
+        margin-left: 25px;
+        font-size: 12pt;
+    }
+    </style>
+
+    <div id="page-wrapper" class="panel-custom" style="background-color: white;">
+        <div style="vertical-align: middle !important;">
+        <div class="row">
+            <h2 class="page-title">WHAT IS THIS?</h2>
```
Comment (dimasciput) [inline flask_project/campaign_manager/templates/about.html]: Don't use all uppercase

File: flask_project/campaign_manager/templates/about.html [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            <div class="page-content">
```
Comment (dimasciput) [inline flask_project/campaign_manager/templates/about.html]: Try to have the same left-padding with the title.

**Comment by ann26** [comment]
ok @dimasciput

**Comment by ann26** [comment]
how about this @dimasciput ?
screencapture-0-0-0-0-5000-campaign_manager-about-1499938810646

**Pull request merged**